app/src/ui/diff/index.tsx

```diff
@@ -952,6 +974,62 @@ export class Diff extends React.Component<IDiffProps, {}> {
     return null
   }
 
+  private renderLargeTextDiff() {
+    return (
+      <div className="panel empty large-diff">
+        <img src={NoDiffImage} />
+        <p>
+          The diff is too large to be displayed by default.
+          <br />
+          You can try to show it anyways, but performance may be negatively
+          impacted.
+        </p>
+        <Button onClick={this.showLargeDiff}>Show diff</Button>
```

Comment:
We should probably case this, ex:
`{__DARWIN__ ? 'Untrusted Server' : 'Untrusted server'}`